astrocut/cutouts.py

```diff
@@ -8,7 +8,7 @@
 
 from time import time
 
-from astropy import log
+from astropy import log as astropy_log
```

Comment:
should this still be using the astropy log?

Reply:
Yes, I think because the `_hducut` function is using the logs from Astropy to determine whether or not a FITS file has SIP keywords in the WCS. I didn't write this code originally, but as far as I can tell from the Astropy source code, there isn't really a way around this without rerouting the logs: https://github.com/astropy/astropy/blob/main/astropy/wcs/wcs.py#L1299